src/libraries/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs

```diff
@@ -28,6 +28,26 @@ namespace System.Net.Http.Functional.Tests
 {
     using Configuration = System.Net.Test.Common.Configuration;
 
+    public class CertificateSetup : IDisposable
+    {
+        public X509Certificate2 ServerCert => _pkiHolder.EndEntity;
+        public X509Certificate2Collection ServerChain => _pkiHolder.IssuerChain;
+
+        Configuration.Certificates.PkiHolder _pkiHolder;
```

Comment:
I think all of the CertificateSetup types are missing an access modifier on the `_pkiHolder` field.

Reply:
Fixed